Doc/whatsnew/3.11.rst

```diff
@@ -493,6 +488,225 @@ Optimizations
+| operations    |                    | such as ``int``, ``float``, and ``str`` take custom   |                   | Dong-hee Na,      |
+|               |                    | fast paths for their underlying types.                |                   | Brandt Bucher,    |
+|               |                    |                                                       |                   | Dennis Sweeney    |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Subscript     | ``o[i]``           | Subscripting container types such as ``list``,        | 10-30%            | Irit Katriel,     |
+|               |                    | ``tuple`` and ``dict`` directly index the underlying  |                   | Mark Shannon      |
+|               |                    | data structures. Subscripting custom ``__getitem__``  |                   |                   |
+|               |                    | is also inlined similar to :ref:`inline-calls`.       |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Store         | ``o[i] = z``       | Similar to subscripting specialization above.         | 10-90%            | Dennis Sweeney    |
+| subscript     |                    |                                                       |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Calls         | ``f(arg)``         | Calls to common builtin (C) functions such as ``len`` | 20%               | Mark Shannon,     |
+|               |                    | and ``isinstance`` directly call their underlying C   |                   | Ken Jin           |
+|               | ``T(arg)``         | version. This avoids going through the internal       | 170%              |                   |
+|               |                    | calling convention.                                   |                   |                   |
+|               |                    |                                                       |                   |                   |
+|               |                    | Calls to certain Python functions are inlined similar |                   |                   |
+|               |                    | to :ref:`inline-calls`.                               |                   |                   |
+|               |                    |                                                       |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Load          | ``print; len``     | The object's index in the globals/builtins namespace  | - [1]_            | Mark Shannon      |
```

Comment:
The `print; len` example makes it think it only applies to builtins. Or that the semicolon is relevant.

Sadly I'm not sure how else to quickly summarize this. Maybe `gbl`, being a random global variable?